.prettierrc.yaml

```
@@ -0,0 +1,9 @@
+# Configuration for the prettier code formatter.
+# https://prettier.io/docs/en/options.html
+tabWidth: 4
+useTabs: false
+semi: false
```

In conformance with the Google TypeScript style guide:
```suggestion
semi: true
```